kolena/workflow/workflow.py

```diff
@@ -90,22 +90,20 @@ def _from_api_response(cls, workflow: str, response: API.EvaluatorResponse) -> "
 
 @dataclass(frozen=True)
 class Workflow:
```

Comment:
I think this class is missing from the doc (or I can't seem to easily find it).

Reply:
I decided to omit it — it's not a useful user-facing class, and it has a few methods on it that it isn't 100% safe to remove but also aren't a part of the "public" API (see #59).

Reply:
Right. That's fair.